templates/rhdm73-authoring-ha.yaml

```diff
@@ -860,8 +865,6 @@ objects:
             value: "${KIE_ADMIN_USER}"
           - name: KIE_ADMIN_PWD
             value: "${KIE_ADMIN_PWD}"
-          - name: KIE_MBEANS
```

Comment:
Is it removed by accident?

Reply:
@sutaakar Nope I just moved it so all the templates have these few parameters in the same order.

Reply:
@sutaakar Oops - yup - you were right. Fixed.